libnavui-androidauto/src/main/java/com/mapbox/androidauto/car/preview/CarRoutePreviewScreen.kt

```diff
@@ -103,14 +95,16 @@ class CarRoutePreviewScreen @UiThread constructor(
 
     init {
         logAndroidAuto("CarRoutePreviewScreen constructor")
+        addBackPressedHandler {
+            logAndroidAuto("CarRoutePreviewScreen OnBackPressedCallback")
+            MapboxNavigationApp.current()!!.setNavigationRoutes(emptyList())
```

Comment:
I think this can be removed because we don't set routes to `MapboxNavigation` in `CarRoutePreviewScreen`.

Reply:
Yeah you're right